backend/db/migrations/env.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```

Comment:
There's a `.pyc` file for this that was also added to the commit. I don't think binaries belong in version control, so we should probably add those to `.gitignore`